index.html

```diff
@@ -92,19 +161,44 @@ <h3 id="the-difficulties-of-ruby-processing">The Difficulties of Ruby Processing
     whether it can be allowed to be laid over the characters preceding or following,
     and whether this affects the position of the base characters</p>
     <aside class=note title="Protrusion of ruby from base characters">
-      In movable type typesetting,
-      ruby characters protruding from their base characters
-      may not hang over neighboring kanji,
-      but often were allowed to hang over neighboring kana.
-      However, when the ruby is katakana,
-      some publishers would set it so that
-      it would not hang over the kana neighboring the base character.
-      Also, when the characters around the base characters
-      were kanji on one side and kana on the other side,
-      for the sake of balance
-      the ruby would sometimes be set so as to hang
-      neither over the kanji nor over the kana
-      (which would therefore both be spaced away from the base character).
+      <span class="checkme">
+      From principle not to have extra spacing between characters in Japanese composition, 
+      one method of layout is to lay over the ruby string preceding or 
+      following Kana characters but not to lay over the ruby string preceding 
+      or following Kanji characters. 
+      With this method, processing may go smoothly for the case with Kana 
+      characters as both preceding and following of ruby base text, 
+      but there is possible case of unbalanced display, 
+      like a combination of Kana character as preceding of ruby base text 
+      and Kanji character as following of ruby base text
```

Comment:
```suggestion
      With this method, it can have an unbalanced look when Kana precedes the
      ruby base and Kanji follows, or vice versa.
```

Simplified.

Reply:
I somehow feel over-simplified on this point,,. How it is unbalanced might be difficult for those who is not familiar with Japanese?

Reply:
```suggestion
      This method works well when characters preceding and following the ruby base are both Kana
      or both Kanji. It however can have an unbalanced look when one is Kana and the other is Kanji
      and if the ruby text is longer than the ruby base.
```

Reply:
I do not think so as it explains that it would lay over Kana but not on Kanji. but I agree it might be over-simplified as it removes a whole sentence of the good case. How about this?

Reply:
updated by afeab5f